android/modules/locale/src/java/ti/modules/titanium/locale/LocaleModule.java

```diff
@@ -106,17 +104,8 @@ public void setLanguage(String language)
 				locale = new Locale(parts[0]);
 			}
 
-			Locale.setDefault(locale);
+			TiLocaleManager.setLocale(locale);
 
-			Configuration config = new Configuration();
-			if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

Comment:
Why was the version guarded as Android N here? setLocale was added in api level 17. Just a mistake, or maybe we had a good reason? @garymathews appears to have been the one to add it, so maybe he knows?

Reply:
It's because the `Configuration` class' public `locale` member variable was deprecated in Android N.
https://developer.android.com/reference/android/content/res/Configuration.html#locale